cnf-certification-test/accesscontrol/rbac/automount_moq_test.go

```diff
@@ -0,0 +1,4 @@
+// Code generated by moq; DO NOT EDIT.
```

Comment:
this file should not be here right ?

Reply:
yep, will delete

Reply:
fixed

Reply:
Probably needs a rebase and a re-run with the updated script from #73